### DISCUSSION TITLE
This issue has not received a response in a while.

### DISCUSSION THREAD

**Discussion by mryellow** [ajaxorg/ace#5111] [General]
Issues don't go away after a few hours when a bot decides to hassle people making them come back to bump issues just to confirm that issues are still issues.

**Comment by oykuyilmaz** [comment]
But do we expect the issues to go away or do you mean preventing bot to stop pinging in case the issue was bumped?

**Comment by mryellow** [comment]
I expect issues to go away when they're solved rather than when a bot hides them.

**Reply by whazor** [reply]
Hi, we feel sorry that you feel rushed by our bot. We added the `no-autoclose` to your issue, and you can take your time to respond.

We actively triage issues and believe our process is good for most issues. If more people share your feeling, we could reconsider the possible response time. For now the overall time is similar to other projects.
